src/I18n/DateFormatTrait.php

```diff
@@ -30,8 +30,17 @@
      * The default locale to be used for displaying formatted date strings.
      *
      * @var string
+     * @deprecated Use self::setDefaultLocale($locale) and self::getDefaultLocale() instead.
```

Comment:
I don't think it makes sense calling the property deprecated when the methods will be actually using it. Instead the docblock should be updated to state that one should use the getter/setter instead of directly manipulating the property.